x-pack/plugins/security_solution/server/lib/entity_analytics/risk_score/painless/risk_scoring_reduce.painless

```diff
@@ -0,0 +1,43 @@
+Map results = new HashMap();
+List inputs = [];
+for (state in states) {
+  inputs.addAll(state.inputs)
+}
+double num_inputs_to_score = Math.min(inputs.length, params.max_risk_inputs_per_identity);
```

Comment:
Should we get rid of this now the sampler limits the number of inputs provided to the risk scoring function?

Reply:
👍 let's remove